sources/install.sh

```diff
@@ -2552,7 +2552,7 @@ function install_wpscan(){
   apt-get install -y apt-transport-https ca-certificates gnupg2 curl
   curl -sSL https://rvm.io/pkuczynski.asc | gpg2 --import -
   curl -sSL https://get.rvm.io | bash -s stable --ruby
-  gem install nokogiri
+  gem install nokogiri -v 1.11.4 -- --use-system-libraries # use this version to resolve the conflict with cewl 
```

Comment:
Add a test command for nokogiri if possible

Reply:
Nokogiri is a lib and I already test tools that has dependencies on.